identifier/src/containers/Login/Login.jsx

```diff
@@ -150,17 +165,17 @@ function Login(props) {
           variant="outlined"
           InputProps={{      // End-adornment icon to show/hide password on visibility button click.
             endAdornment: <InputAdornment position="end">
-              <IconButton
-                aria-label="toggle password visibility"
-                onClick={() => setShowPassword((show) => !show)}
-                onMouseDown={(event) => {
-                  event.preventDefault();
-                }}
-                edge="end"
-              >
-                {showPassword ? <VisibilityOff /> : <Visibility />}
-              </IconButton>
-            </InputAdornment>
+                <IconButton
+                  aria-label="toggle password visibility"
+                  onClick={() => setShowPassword((show) => !show)}
+                  onMouseDown={(event) => {
+                    event.preventDefault();
+                  }}
+                  edge="end"
+                >
+                  {showPassword ? <VisibilityOff /> : <Visibility />}
+                </IconButton>
+              </InputAdornment>
```

Comment:
Now there are lot of unrelated white-space changes in this file. Please play attention to your commits and what you push to avoid such changes in the future. You have to undo them.